po/vi.po

```diff
@@ -1837,6 +1966,11 @@ msgid ""
 "get going. Translations are coordinated on the [issue tracker](https://"
 "github.com/google/comprehensive-rust/issues/282)."
 msgstr ""
+"Nếu bạn muốn đóng góp giúp đỡ, vui lòng xem [hướng dẫn của chúng tôi]"
+"(https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md) để "
+"biết nên bắt đầu như thế nào. Các bản dịch được hợp tác cùng hoàn thành trên "
+"một bản [issue tracker](https://github.com/google/comprehensive-rust/"
```

Comment:
The issue is rather outdated. Can you replace it with https://github.com/google/comprehensive-rust/issues/1957 instead?